functions.py

```diff
@@ -0,0 +1,42 @@
+# functions for interaction with POSTGRESQL server, file hosting server and push notifications server
+
+'''from models import Users, Posts, Documents, Forms''' 		# import datatables from postgresql database
+
+def get_posts(get_archived=False):
+	if get_archived:
+		archived_posts = Posts.query.filter_by(archived=True).order_by(desc(Posts.uploadtime)).all()			# getting all ordered archived posts
+		return archived_posts
+	else:
+		unarchived_posts = Posts.query.filter_by(archived=False).order_by(desc(Posts.uploadtime)).all()			# getting all ordered unarchived posts
+		return unarchived_posts
+
+def create_post(title, text, author):																			# creating a new post from backend
+	#Processing data
```

Comment:
I don't know why the original authors did this originally, and it's honestly quite terrible. We should figure out why this was needed. If we keep it, I suggest doing something like:

```python
def rep(s):
    for k, v in {'“':'"', '”':'"', "’":"'"}.items():
        s.replace(k, v)

title = rep(title)
text = rep(text)
author = rep(author)
```

^I haven't tested it, so don't just copy paste it. The benefit is that the same line of code is not repeated three times.